Mellanox/ufm_sdk_3.0
# Issue:3950848:max_num_isolation must be verified against each isolation and not be a bulk

## What
Fix check for maximum number of isolated ports.

## Why ?
Current implementation checks number of detected issues instead of real number of isolated ports.

## Testing ?
1. Set maximum number 2 in /config/pdr_deterministic.conf (MAX_NUM_ISOLATE=2)
2. Run simulated test
3. Check logs (additional ports have not been isolated if there are already 2 isolated ports)

## Special triggers
_Use the following phrases as comments to trigger different runs_

* ```bot:retest``` rerun Jenkins CI (to rerun GitHub CI, use "Checks" tab on PR page and rerun _all_ jobs)
* ```bot:upgrade``` run additional update tests
